idl/chromadb/proto/coordinator.proto

```diff
@@ -1,8 +1,9 @@
 syntax = "proto3";
 
 package chroma;
+option go_package = "github.com/chroma/chroma-coordinator/internal/proto/coordinatorpb";
 
-import "chromadb/proto/chroma.proto";
+import "chroma.proto";
```

Comment:
I think this may break the python code gen, can you update your proto path to work with it as is?